packages/node_modules/@webex/internal-plugin-conversation/src/share-activity.js

```diff
@@ -257,8 +259,8 @@ const ShareActivity = WebexPlugin.extend({
   _upload(file, uri, uploadOptions) {
     const fileSize = file.length || file.size || file.byteLength;
     const fileHash = sha256(file).toString();
-    const {role} = uploadOptions ?? {};
-    const initializeBody = Object.assign({fileSize}, role && {role});
+    const {role, claimedFileType} = uploadOptions ?? {};
+    const initializeBody = Object.assign({fileSize}, {claimedFileType}, role && {role});
```

Comment:
Non-blocking.

Could this be written as:

```js
initializBody = { fileSize, claimedFileType, role };
```

?

Reply:
> These changes look good. We noticed that this is marked as a `non-breaking change`, does this negatively impact any existing requests due to the new `claimedFileType` key?

Not any negative impact on existing request due the new changes.

Reply:
> Non-blocking.
> 
> Could this be written as:
> 
> ```js
> initializBody = { fileSize, claimedFileType, role };
> ```
> 
> ?

Yes we could do that, but the main reason to wrote it like this is because we are avoiding the case of adding 
`role: undefined` to initializeBody object. If we wrote `initializeBody = { fileSize, claimedFileType, role };`  and role is undefined (that would be often because it's added only on avatar change) we will get something like this:

`{fileSize: 5727, claimedFileType: '.zip', role: undefined}` so the `role: undefined` will be added to the initializeBody object

 and with `const initializeBodyy = Object.assign({fileSize}, {claimedFileType}, role && {role});` we are getting this:
 `{fileSize: 5727, claimedFileType: '.zip'}` so we are avoiding to assign role to object if it is undefined.